### PR TITLE
93 entwicklerdokumentation fuer wfd

### PR DESCRIPTION
**Pull request by Schlangi**
Add overview in README.md with links to technical and non-technical docs, add step-by-step commands to install EPS and add some troubleshooting hints.

### CODE REVIEW

File: connect_your_app_to_IRIS/technical_details/app_onboarding.md [hunk 1]
```diff
@@ -94,15 +216,66 @@ Sending messages to IRIS is done with JSON-RPC.
...
+
+It is important to know, what is going wrong if you won't get this response. If you post your request to a wrong endpoint, you would just get a "message":"not found". If the port 5556 is not bound to the eps container, you will get a "connection refused".
+
+| cURL result | Possible cause | Ckecks to perform |
+|-|-| - |
+| Client sent an HTTP request to an HTTPS server. | You called the endpoint at http:// instead of https:// | Ckeck the address in cURL command and options you used. |
+| Connection refused | The EPS is not listening on the URI or port of the request. | Ckeck the address in cURL command. Are you running the command from localhost - the docker host, or do you need to address an IP address in your network? Is there a firewall blocking the port? Is the EPS running - check it with docker ps, and check the logs. |
+| "message":"not found" | You addressed wrong endpoint | If you address https://localhost:5556/ (without endpoint address jsonrpc) you will get this result. |
+| {"jsonrpc":"2.0","error":{"code":-32603,"message":"internal error"},"id":"1"} | There is a bug in EPS or your configuration is incorrect. | Check the settings yml, try another EPS version. |
```
Comment (lucky-lusa) [inline connect_your_app_to_IRIS/technical_details/app_onboarding.md:252]: I welcome the fact that we are dealing with the possible errors. We can go into more detail here at a later date.

File: connect_your_app_to_IRIS/technical_details/app_onboarding.md [hunk 2]
```diff
@@ -347,9 +520,9 @@ AttendanceInformation object:
 | `attendTo` | Attend to | true |
 | `additionalInformation` | Additional attendance information | false | For example table or area 
 
-## Test your implementation
+## 5 Test your implementation
 
-To test your implementation, visit [https://iris.staging.iris-gateway.de](https://iris.staging.iris-gateway.de) 
+To test your implementation, visit [https://iris.staging.iris-gateway.de:9443](https://iris.staging.iris-gateway.de)
```
Comment (lucky-lusa) [inline connect_your_app_to_IRIS/technical_details/app_onboarding.md:525]: Without port 9443 is okay, we have switched.

**Review by lucky-lusa — COMMENTED** [review]
Thank you so much, the documentation is so much more detailed and easier to understand!

File: connect_your_app_to_IRIS/technical_details/app_onboarding.md (hunk 2 above)
Comment (Schlangi) [reply connect_your_app_to_IRIS/technical_details/app_onboarding.md:525]: Yeah, link was already changed but link text left unchanged. In this case a link text is useless, so I changed to just the URL.

**Comment by topialla** [comment]
can we merge this @lucky-lusa @stefanmichalk ?

**Comment by stefanmichalk** [comment]
Yes please!

**Pull request merged**